packages/xo-web/src/common/xo/index.js

```diff
@@ -950,6 +950,8 @@ const chooseActionToUnblockForbiddenStartVm = props =>
 
 const cloneAndStartVm = async (vm, host) => _call('vm.start', { id: await cloneVm(vm), host: resolveId(host) })
 
+const _startVm = (id, host, force = false) => _call('vm.start', { id, host, force })
```

Comment:
`checkMacAddresses` param is missing. You can also use named params to not have to pass `force` when it isn't necessary.